pkgs/development/libraries/glibc/default.nix

```diff
@@ -153,5 +154,6 @@ callPackage ./common.nix { inherit stdenv; } {
 
```

Comment:
the preInstall on line 77 is now running instead of being nulled when build != host

https://github.com/NixOS/nixpkgs/blob/0de5723451490540e7f6a9b6d1015f8f1b04bfc8/pkgs/development/libraries/glibc/common.nix#L270-L272


it was added in 
https://github.com/nixos/nixpkgs/commit/25edc476fd9fe1bd8bedf571d218ba4f27fb5a27


move the conditional to the preInstall